app/screens/home-screen/home-screen.tsx

```diff
@@ -453,6 +460,26 @@ export const HomeScreen: React.FC = () => {
     </Modal>
   )
 
+  React.useEffect(() => {
```

Comment:
we already discuss this in the original PR... this is unnecessary and/or we need to find an alternative